### PR TITLE
refactor: update ts and notification about interop

### PR DESCRIPTION
**Pull request by zxyao145**

### CODE REVIEW

File: components/core/JsInterop/interop.ts [hunk 1]
```diff
@@ -3,10 +3,20 @@ export function getDom(element) {
         element = document.body;
     } else if (typeof element === 'string') {
         element = document.querySelector(element);
+    } else {
+        element = null;
```
Comment (ElderJames) [inline components/core/JsInterop/interop.ts:7]: Why null? If you pass in an ElementReference from C#,js should get the dom.

File: components/core/JsInterop/interop.ts [hunk 2]
```diff
@@ -169,61 +179,216 @@ export function delElementFrom(delElement, elementSelector) {
     getDom(elementSelector).removeChild(delElement);
 }
 
-/************ Notification start ************/
-function getNotificationService() {
-    return (<any>window).notificationService;
-}
-
+/**
+ * ��html�ַ�תΪelement
```
Comment (ElderJames) [inline components/core/JsInterop/interop.ts:183]: Try to use English.

File: components/core/JsInterop/interop.ts [hunk 3]
```diff
@@ -169,61 +179,216 @@ export function delElementFrom(delElement, elementSelector) {
...
+    }
+    csInstance[instanceName] = instance;
+}
+
+/**
+ * ɾ��js��ע���C#���ʵ������
+ * @param instanceName
+ */
+export function delCsInstance(instanceName: string) {
+    let csInstance = (<any>window).antBlazor.interop.csInstance;
```
Comment (ElderJames) [inline components/core/JsInterop/interop.ts:337]: In TS, you can declare `notificationService` and `csInstance ` as the export module.
Don't define it directly on `window`.

File: components/core/JsInterop/interop.ts (hunk 1 above)
Comment (zxyao145) [reply components/core/JsInterop/interop.ts:7]: sorry, I forgot it.

File: components/core/JsInterop/interop.ts (hunk 2 above)
Comment (zxyao145) [reply components/core/JsInterop/interop.ts:183]: OK, I will try to use English.

File: components/core/JsInterop/interop.ts (hunk 3 above)
Comment (zxyao145) [reply components/core/JsInterop/interop.ts:337]: OK, I'll change it.

File: components/core/JsInterop/interop.ts [hunk 4]
```diff
@@ -169,61 +179,216 @@ export function delElementFrom(delElement, elementSelector) {
...
-function removeEnterClass(element) {
-    let newClassName = element.getAttribute("class").split(" ").map((item) => {
+/**
+ * ��html׷�ӵ�selector��ĩβ
+ * @param html  string(cssѡ����) | Node
+ * @param containerSelector string(cssѡ����) | HTMLElement
+ */
+export function appendHtml(html: string | Node,
+    containerSelector: string | HTMLElement = null) {
+    console.log("appendHtml,containerSelector", containerSelector);
```
[the remark this answers is not in the document]
Comment (zxyao145) [reply components/core/JsInterop/interop.ts]: In the latest commit, I've removed it.

**Comment by zxyao145** [comment]
code rebase error, and it will be a new PR.

**Pull request closed without merging**